sdk/clientcore/core/src/samples/java/io/clientcore/core/http/annotation/AnnotationJavadocCodesnippets.java

```diff
@@ -202,8 +202,8 @@ VirtualMachine getByResourceGroup(@PathParam("resourceGroupName") String rgName,
         // END: io.clientcore.core.http.annotation.Host.class1
 
         // BEGIN: io.clientcore.core.http.annotation.Host.class2
-        @ServiceInterface(name = "KeyVaultService", host = "https://{vaultName}.vault.azure.net:443")
-        interface KeyVaultService {
+        @ServiceInterface(name = "SecretsService", host = "https://{instance}.somecloud.com:443")
```

Comment:
Need to update the corresponding `@HostParam` name, otherwise this snippet is no longer valid code